lahja/endpoint.py

```diff
@@ -641,8 +705,7 @@ def broadcast_nowait(self,
         should be broadcasted to. By default, requests are broadcasted across all connected
         endpoints with their consuming call sites.
         """
-        item._origin = self.name
-        item._id = str(uuid.uuid4())
+        item.bind(self, str(uuid.uuid4()))
```

Comment:
TODO, I think the `id` should potentially be passed into `broadcast` to remove the *weirdness* of this pre-binding the event, and thus `broadcast` having to check if the event is already bound.